### PR TITLE
(docs) O3-1963: Update `yarn start` command in Readme

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

It fixes the docs for setting up the project in the readme.

## Related Issue

https://issues.openmrs.org/browse/O3-1963

### CODE REVIEW

**Review by Piumal1999 — APPROVED** [review]
Good job! Thank you @ayush-AI 

Please include the JIRA issue link in your pull request description

Edit: You can follow this pull request description template: https://github.com/openmrs/openmrs-esm-patient-management/blob/main/.github/pull_request_template.md

**Comment by ayush-AI** [comment]
> Good job! Thank you @ayush-AI
> 
> Please include the JIRA issue link in your pull request description
> 
> Edit: You can follow this pull request description template: https://github.com/openmrs/openmrs-esm-patient-management/blob/main/.github/pull_request_template.md

Thanks, I updated the description.

**Review by ibacher — CHANGES_REQUESTED** [review]
Please don't suggest this. I'm happy to have an example, e.g., with `yarn start --sources packages/esm-appointments-app`, but this is suggesting that the default should be to start all the microfrontends locally, which it shouldn't.

**Comment by ayush-AI** [comment]
@ibacher update the PR with the example as you suggested. Please have a look and let me know if anything else need to be updated.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ To start a dev server running all the modules simultaneously, run:
 ```bash
 yarn start
 ```
+Note: You need to specify the sources for yarn start to work.For example - `yarn start --sources 'packages/esm-patient-search-app'`
```
Comment (kdaud) [inline README.md:43]: This should be true when running a specific module but not for `yarn start`. Did you see [this](https://github.com/openmrs/openmrs-esm-patient-management#:~:text=To%20start%20a%20dev%20server%20for%20a%20specific%20module%2C%20run%3A)?

Comment (ayush-AI) [reply README.md:43]: Hello @kdaud yes I already saw that. But yarn start will always throw error if we do not specify the sources. Initially I modified the `yarn start` command to `yarn start --sources packages/esm-*-app` but I modified the docs again after the suggestions. Any suggestion on how exactly should we put this in documentation will be a great help. 
Thanks for reviewing.

Comment (kdaud) [reply README.md:43]: Could you run again the command `yarn start`?

Comment (ayush-AI) [reply README.md:43]: I tried but getting [error](https://pastebin.com/mrGRQM16).

Comment (jayasanka-sack) [reply README.md:43]: Yes `yarn start` won't work without --sources
 
Maybe we can have something similar to https://github.com/openmrs/openmrs-esm-patient-chart

Comment (ayush-AI) [reply README.md:43]: yes, I guess that will be optimal solution for the issue. Should I go ahead and update the PR and make it similar to this?

Comment (jayasanka-sack) [reply README.md:43]: yes. Go ahead.

File: README.md [hunk 2]
```diff
@@ -35,22 +35,14 @@ To set up environment variables for the project, follow these steps:
...
-
-Note that this is very resource-intensive.
-
 To start a dev server for a specific module, run:
 
 ```bash
 yarn start --sources 'packages/esm-<insert-package-name>-app'
 ```
 
+This command uses the [openmrs](https://www.npmjs.com/package/openmrs) tooling to fire up a dev server running `esm-patient-chart` as well as the specified module.
```
Comment (ayush-AI) [inline README.md:44]: Here the name will be esm-patient-chart or esm-patient-management ?

**Comment by ayush-AI** [comment]
Updated the PR with required changes. Please have a look @jayasanka-sack @ibacher @kdaud @Piumal1999

File: README.md (hunk 2 above)
Comment (jayasanka-sack) [reply README.md:44]: It should be `esm-patient-management`

Comment (ayush-AI) [reply README.md]: Changed it. Please let me know if anything more need to be updated.

**Review by vasharma05 — COMMENTED** [review]
Looks good to me!

@ibacher please give the final approval! Thanks!

**Comment by ayush-AI** [comment]
@ibacher If anything need to be updated. Please let me know.

**Review by ibacher — APPROVED** [review]

**Pull request merged**